### ISSUE TITLE
Wiki template

### ISSUE THREAD

**Issue by chekoopa** [bouncepaw/mycorrhiza#55]
A little UX quality-of-life improvement: offer user to create an empty template wiki by a separate argument, such as `-template`:

```
2021/05/17 16:37:24 Error: pass a wiki directory
2021/05/17 16:37:24 Error: or run mycorrhiza -template <path>
```

The template command would `git init` a new wiki and (preferrably) set it to support Unicode paths. As an ideal, it would also left a template home hypha with some tips on next configuration steps (such as run `mycorrhiza -print-example-config` etc)

**Comment by bouncepaw** [comment]
I've had an idea of a wizard that would ask the following questions:

1. Name of the wiki?
2. URL of the wiki?
3. Use registration?
4. etc

Perhaps it would be more useful than such templates.

I also want to create an example wiki (https://github.com/bouncepaw/metarrhiza doesn't seem to be a good actor for this role). Such example wiki could work like templates.

**Comment by DanInSpace104** [comment]
> I've had an idea of a wizard that would ask the following questions:

> Perhaps it would be more useful than such templates.

I think it is a bit different case, cause templates can be a lot more complex than just configs and home page. For example, I can make template for diary with empty pages for every day in next year and then share it with my friends (not the best example)).

**Comment by bouncepaw** [comment]
> I think it is a bit different case, cause templates can be a lot more complex than just configs and home page. For example, I can make template for diary with empty pages for every day in next year and then share it with my friends

Hmm, indeed. If there are different templates, how are they made? Git repos with hyphae? Scripts that create hyphae through HTTP?

**Comment by handlerug** [comment]
Even such a basic flag that would set sensible default would've been very useful for me. I had to figure out all issues with Unicode and stuff by myself, and I like this idea, even if we keep it simple for now.

**Comment by handlerug** [comment]
Might even assign myself to this issue and maybe implement a wizard 👀

**Comment by bouncepaw** [comment]
What a great human you are ❤️ Please don't forget to document the stuff, I've opened registration on https://mycorrhiza.lesarbr.es/register

**Comment by GuAlSe** [comment]
> Hmm, indeed. If there are different templates, how are they made?

I think git repositories is perfect. Wizard can have several templates and clone needed or create blank wiki.

**Comment by bouncepaw** [comment]
> Wizard can have several templates and clone needed or create blank wiki.

Yeah, this approach is also decentralised (more or less). Any person can host a git repository, thus, any person can hot a wiki template.

**Comment by GuAlSe** [comment]
My approach wizard example:

```
Welcome to MycorrhizaWiki wizard. Please select the template you want:
1. Home wiki
2. Daily
3. Public wiki
0. No from there, I want to create wiki from scratch.
[0-3]: 0
Okay, what name of it? MyWiki
Is it need registration? [y/n] yep
…
```

**Comment by handlerug** [comment]
By the way, @chekoopa, your original feature request has been implemented in 1.2 with the introduction of new wiki directory structure. No need to use a `-template` flag, though, all you need to do is to run `mycorrhiza path/to/wiki` and the repo will be automatically initialized with recommended settings.
